scripts/upload-s3.sh

```diff
@@ -1,3 +1,6 @@
 #!/bin/sh
 
+# Remove any files that could contain search results from a search API
+grep -c "URL:" "${TMPDIR}"/gptscript-state* | grep -v ":0" | cut -d ':' -f1 | xargs rm
```

Comment:
Should we be worried if other response data could have `URL:` that's not actually from search tool?